runtime/calamari/src/lib.rs

```diff
@@ -752,8 +780,50 @@ pub type Executive = frame_executive::Executive<
     frame_system::ChainContext<Runtime>,
     Runtime,
     AllPalletsReversedWithSystemFirst,
-    (),
+    UpgradeSessionKeys,
```

Comment:
Please run the try-runtime GHA

Reply:
Completed here: https://github.com/Manta-Network/Manta/actions/runs/2634849527